### PR TITLE
Partial Filipino Translation  - 1 119 translated words

### PR DESCRIPTION
**Pull request by mind-blast**
Translating the file : rewriting-history.asc with total translated file : 1119 translated words
Note : Code text and computer code text are not included

### CODE REVIEW

File: book/07-git-tools/sections/rewriting-history.asc [hunk 1]
```diff
@@ -8,41 +8,41 @@ Maaaring sangkot ang pagbabagot ng pagkasunod-sunod ng mga commit, pagbabago ng
...
 Sa seksyon ito, makikita mo kung paaano gagawin ang mga gawaing ito upang makagawa ka ng iyong kasaysayan na magmukha sa gusto mong unang iyong ibinahagi sa iba.
 
-[NOTE]
+[TANDAAN]
 ====
-One of the cardinal rules of Git is that, since so much work is local within your clone, you have a great deal of freedom to rewrite your history _locally_.
-However, once you push your work, it is a different story entirely, and you should consider pushed work as final unless you have good reason to change it.
-In short, you should avoid pushing your work until you're happy with it and ready to share it with the rest of the world.
+Isa sa kardinal na mga patakaran sa Git ay ang, mula ng maraming trabaho ay ang lokal na nasa loob ng iyong clone, meron kang isang magandang pakikitungo sa kalayaan upang sumulat muli ng iyong kasaysayan ng _naka-lokal_.
+Gayunpaman, Minsan ikaw ang nag-push ng iyong trabaho, ito ay isang ibang kwento sa kabuuan, at ikaw ay dapat kilalanin ang na-push na trabaho bilang huli maliban kung ikaw ay merong magandang dahilan upang baguhin ito.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: Minsan - minsan

File: book/07-git-tools/sections/rewriting-history.asc [hunk 2]
```diff
@@ -8,41 +8,41 @@ Maaaring sangkot ang pagbabagot ng pagkasunod-sunod ng mga commit, pagbabago ng
...
+Sa madaling salita, ikaw ay dapat umiwas sa pag-push ng iyong tranaho hanggang ikaw ay masaya nito at handang ibahagi sa lahat sa mundo.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: > rest of the word

could also be translated to:

> sa buong mundo
_(simplified)_

File: book/07-git-tools/sections/rewriting-history.asc [hunk 3]
```diff
@@ -8,41 +8,41 @@ Maaaring sangkot ang pagbabagot ng pagkasunod-sunod ng mga commit, pagbabago ng
...
 ====
 
 [[_git_amend]]
-==== Changing the Last Commit
+==== Pagbabago sa Huling Commit
 
-Changing your most recent commit is probably the most common rewriting of history that you’ll do.
-You’ll often want to do two basic things to your last commit: simply change the commit message, or change the actual content of the commit by adding, removing and modifying files.
+Pagbabago ng iyong pinakabagong commit ay marahil nagsusulat muli ng kasaysayan na gagawin mo.
+Madalas mong gustong gawin ang dalawang pangunahing mga bagay sa iyong huling commit: baguhin lang ang commit na mensahe, o baguhin ang aktwal na laman sa commit sa pamamagitan ng pag-add, pag-alis at pagbago ng mga file.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: pag-add to pagdaragdag

File: book/07-git-tools/sections/rewriting-history.asc [hunk 4]
```diff
@@ -8,41 +8,41 @@ Maaaring sangkot ang pagbabagot ng pagkasunod-sunod ng mga commit, pagbabago ng
...
-When you save and close the editor, the editor writes a new commit containing that updated commit message and makes it your new last commit.
+Ang utos sa itaas ay nagkakarga ng mga nakaraang commit na mensahe sa editor na sesyon, kapag ikaw ay gumawa ng mga pagbabago sa mensahe, i-save ang mga pagbabagong iyon at lumabas.
+Kapag ikaw ay nag-save at nagsara sa editor, ang editor ay nagsusulat ng isang bagong commit na naglalaman sa pinakabago na commit na mensahe at ginagawa itong iyong bagong huling commit.
 
-If, on the other hand, you want to change the actual _content_ of your last commit, the process works basically the same way -- first make the changes you think you forgot, stage those changes, and the subsequent `git commit --amend` _replaces_ that last commit with your new, improved commit.
+Kung, sa kabilang banda, ikaw ay gusto magbago ng aktwal na _nilalaman_ sa iyong huling commit, ang proseso ay gumagana talaga sa parehong paraan -- una gumawa ng mga pagbabago na sa tingin mo ay nakalimutan, i-stage ang mga pagbabagong iyon, at ang kasunod na `git commit --amend` _ay napapalit_ sa huling commit na iyon sa iyong bago, na pinabuting commit.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: gusto magbago to gusto**ng** magbago

File: book/07-git-tools/sections/rewriting-history.asc [hunk 5]
```diff
@@ -8,41 +8,41 @@ Maaaring sangkot ang pagbabagot ng pagkasunod-sunod ng mga commit, pagbabago ng
...
 ====
-When you amend a commit, you have the opportunity to change both the commit message and the content of the commit.
-If you amend the content of the commit substantially, you should almost certainly update the commit message to reflect that amended content.
+Kapag iyong binago ang commit, mayroon kang pagkakataon na baguhin ang kapwa commit na mensahe at ang nilalaman sa commit.
+Kung ikaw ay nagbago sa nilalaman sa commit na kalahatan, ikaw ay dapat halos tiyak sa pagbago sa commit sa mensahe upang sumasalamin sa nabago na nilalaman.
 
-On the other hand, if your amendments are suitably trivial (fixing a silly typo or adding a file you forgot to stage) such that the earlier commit message is just fine, you can simply make the changes, stage them, and avoid the unnecessary editor session entirely with:
+Sa kabilang banda, kung ang iyong pagbabago ay nagaangkop ng walang halaga (pag-ayos sa isang hangal na pag-type o pagdagdag ng isang file na iyong nakalimutan ma-stage) tulad ng sa mas maagang commit na mensahe ay mabuti lang, magagawa mo lang na baguhin, i-stage sila, at iwasan ang hindi kinakailangan na sesyon ng editor na ganap na may:
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: nagaangkop to nag-aangkop

File: book/07-git-tools/sections/rewriting-history.asc [hunk 6]
```diff
@@ -52,26 +52,26 @@ $ git commit --amend --no-edit
...
-==== Changing Multiple Commit Messages
+==== Pagbabago sa Maramihang Commit na mga Mensahe
 
-To modify a commit that is farther back in your history, you must move to more complex tools.
-Git doesn’t have a modify-history tool, but you can use the rebase tool to rebase a series of commits onto the HEAD they were originally based on instead of moving them to another one.
-With the interactive rebase tool, you can then stop after each commit you want to modify and change the message, add files, or do whatever you wish.
-You can run rebase interactively by adding the `-i` option to `git rebase`.
-You must indicate how far back you want to rewrite commits by telling the command which commit to rebase onto.
+Upang baguhin ang isang commit na iyon na mas malayo sa iyong kasaysayan, ikaw ay dapat lumipat sa higit na kumplikadong mga kasangkapan.
+Ang Git ay walang pagbabago-kasaysayan na kasangkapan, ngunit maaari mong gamitin ang rebase na kasankapan sa mga serye ng mga commit papunta sa HEAD sila ay orihinal batay sa halip na paglipat sa kanila na para sa isa pa.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: kasankapa to kasangkapan

File: book/07-git-tools/sections/rewriting-history.asc [hunk 7]
```diff
@@ -52,26 +52,26 @@ $ git commit --amend --no-edit
...
+Gamit ang interactuve na rebase na kasangkapan, maaari mong itigil pagkatapos ng bawat commit na gusto mong baguhin at palitan ang mensahe, magdagdag ng mga file, o gumawa ng gusto mong gawin.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: interactuve > interactive

File: book/07-git-tools/sections/rewriting-history.asc [hunk 8]
```diff
@@ -52,26 +52,26 @@ $ git commit --amend --no-edit
...
+Maaari mong patakbuhin ang rebase nang interactive sa pamamagitan sa pagdagdag ng `-i` na opsyon sa `git rebase`.
+Dapat mong ipahiwatig kung gaano kalayo ang gusto mong isulat muli sa mga commit sa pamamagitan sa pagsabi ng utos na kung saan ang commit ay nag-rebase nito.
 
-For example, if you want to change the last three commit messages, or any of the commit messages in that group, you supply as an argument to `git rebase -i` the parent of the last commit you want to edit, which is `HEAD~2^` or `HEAD~3`.
-It may be easier to remember the `~3` because you’re trying to edit the last three commits, but keep in mind that you’re actually designating four commits ago, the parent of the last commit you want to edit:
+Halimbawa, kung ikaw ay gustong baguhin ang huling tatlong commit na mga mensahe, o anuman sa mga commit na mensahe sa grupong iyon, punan mo yun bilang isang argumento sa `git rebase -i` at ang magulang sa huling commit na gusto mong baguhin, na kung saan ay ang `HEAD~2^` o `HEAD~3`.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: > kung ikaw ay gustong baguhin ang huling tatlong commit na mga mensahe

could be

> kung gusto mong baguhin ang huling tatlong commit na mga mensahe

File: book/07-git-tools/sections/rewriting-history.asc [hunk 9]
```diff
@@ -52,26 +52,26 @@ $ git commit --amend --no-edit
...
+Maaaring mas madali itong matandaan ang `~3` bdahil ikaw ay sinusubukang mag-edit sa huling tatlong mga commit, ngunit panataliin sa isip na ikaw ay tunay na nagtatalaga ng apat na mga commit nung nakaraan, ang magulang sa huling commit na gusto mong i-edit:
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: bdahil > dahil

File: book/07-git-tools/sections/rewriting-history.asc [hunk 10]
```diff
@@ -52,26 +52,26 @@ $ git commit --amend --no-edit
...
 
 [source,console]
 ----
 $ git rebase -i HEAD~3
 ----
 
-Remember again that this is a rebasing command – every commit included in the range `HEAD~3..HEAD` will be rewritten, whether you change the message or not.
-Don’t include any commit you’ve already pushed to a central server – doing so will confuse other developers by providing an alternate version of the same change.
+Tandaan muli na ito ay isang rebase na utos – Sa bawat commit na kasama sa lawak ng `HEAD~3..HEAD` ay masusulatang muli, kung ikaw ay nagbago sa mensahe o hindi.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: * Tandaan muli to Tandaang muli o Muling tandaan
* Sa bawat > sa bawat

File: book/07-git-tools/sections/rewriting-history.asc [hunk 11]
```diff
@@ -52,26 +52,26 @@ $ git commit --amend --no-edit
...
+Hindi nagsasali ng anumang commit na iyong na-push na sa sentral server – paggawa nito ay malilito ang ibang mga developer sa pamamagitan ng pagbigay ng isang alternatibong bersyon sa parehong pagbabago.
 
-Running this command gives you a list of commits in your text editor that looks something like this:
+Pagtatakbo nitong utos ay nagbigigay sayo ng listahan ng mga coomit sa iyong text editor na tulad nito:
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: coomit to commit

File: book/07-git-tools/sections/rewriting-history.asc [hunk 12]
```diff
@@ -98,8 +98,8 @@ pick a5f4a0d added cat-file
 # Note that empty commits are commented out
 ----
 
-It’s important to note that these commits are listed in the opposite order than you normally see them using the `log` command.
-If you run a `log`, you see something like this:
+Ito ay importanteng tandaan na ang mga commit na ito ay nakalista sa kabaligtaran na pagkasunod-sunod kaysa sa normal na nakikita sa paggamit ng `log` na utos.
+Kung ikaw ay nagpapatakno ng `log`, ikaw ay makakita ng isang bagay na tulad nito:
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: nagpapatakno > nagpapatakbo

File: book/07-git-tools/sections/rewriting-history.asc [hunk 13]
```diff
@@ -109,14 +109,14 @@ a5f4a0d added cat-file
 f7f3f6d changed my name a bit
 ----
 
-Notice the reverse order.
-The interactive rebase gives you a script that it’s going to run.
-It will start at the commit you specify on the command line (`HEAD~3`) and replay the changes introduced in each of these commits from top to bottom.
-It lists the oldest at the top, rather than the newest, because that’s the first one it will replay.
+Pansinin ang baligtad na pagkasunod-sunod.
+Ang interactive na rebase ay nagbibigay sa iyo ng isang script na ito ay tatakbo.
+Ito ay nag-uumpisa sa commit na iyong itinukoy sa command line (`HEAD~3`) at i-replay ang mgapababago na ipinakilala sa bawat mga commit nito mula sa taas hanggang sa ibaba.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: mgapababago - space

File: book/07-git-tools/sections/rewriting-history.asc [hunk 14]
```diff
@@ -109,14 +109,14 @@ a5f4a0d added cat-file
...
+Ito ay naglilista sa pinakamatanda sa itaas, sa halip na ang pinakabago, dahil iyong ang unang mai-reply.
 
-You need to edit the script so that it stops at the commit you want to edit.
-To do so, change the word `pick' to the word `edit' for each of the commits you want the script to stop after.
-For example, to modify only the third commit message, you change the file to look like this:
+Ikaw ay kailangang mag-edit sa script upang tumigil ang commit na gusto mong i-edit.
+Upang gawin ito, baguhin ang salitang `pmili' sa salitang `edit' para sa bawat mga commit na gusto mo na script upang itigil ito pagkatapos.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: pmili > pumili

File: book/07-git-tools/sections/rewriting-history.asc [hunk 15]
```diff
@@ -140,30 +140,30 @@ Once you’re satisfied with your changes, run
...
+Uri
 
 [source,console]
 ----
 $ git commit --amend
 ----
 
-Change the commit message, and exit the editor.
-Then, run
+Paguhin ang commit na mensahe, at lumabas ang editor.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: Paguhin > Baguhin

File: book/07-git-tools/sections/rewriting-history.asc [hunk 16]
```diff
@@ -140,30 +140,30 @@ Once you’re satisfied with your changes, run
...
+Ang utos na ito ay mag-apply sa ibang dalawang mga commit na awtomatiko, at pagkatapos ay ikaw ay natapos na.
+Kung ikaw ay nagpalit ng pagkuha sa pag-edit sa maraming mga linya, maaari mong ulitin ang mga hakbang na ito sa bawat commit na iyong binago sa edit.
+Bawat oras, ang Git ay hihinto , hayaan mong baguhin ang commit, at patuloy kapag ikaw ay natapos.
 
-==== Reordering Commits
+==== Pagsunod-sunod ng mga Commit
 
-You can also use interactive rebases to reorder or remove commits entirely.
-If you want to remove the ``added cat-file'' commit and change the order in which the other two commits are introduced, you can change the rebase script from this
+Maaari mong ring gamitin din ang interactive na mga rebase upang isunod-sunod o tanggalin ang mga commit kabuuan.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: Maaari mong ring gamitin ~~din~~

File: book/07-git-tools/sections/rewriting-history.asc [hunk 17]
```diff
@@ -172,22 +172,22 @@ pick 310154e updated README formatting and added blame
...
 ----
 
-When you save and exit the editor, Git rewinds your branch to the parent of these commits, applies `310154e` and then `f7f3f6d`, and then stops.
-You effectively change the order of those commits and remove the ``added cat-file'' commit completely.
+Kapag ikaw ay nag-save at lumabas sa editor, ang Git ay nag-uulit ng iyong branch sa magulang ng mga commit, naaangkop sa `310154e` at pagkatapos ang `f7f3f6d`, at pagkatapos ay huminto.
+Ikaw ay epektibong nagbago sa pagkasunod-sunod ng mga commit at alisin ang ``added cat-file'' ang commit na ganap.
 
 [[_squashing]]
-==== Squashing Commits
+==== Pag-squashng mga  Commit
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: Pag-squashng - space

File: book/07-git-tools/sections/rewriting-history.asc [hunk 18]
```diff
@@ -209,8 +209,8 @@ The script puts helpful instructions in the rebase message:
 # Note that empty commits are commented out
 ----
 
-If, instead of ``pick'' or ``edit'', you specify ``squash'', Git applies both that change and the change directly before it and makes you merge the commit messages together.
-So, if you want to make a single commit from these three commits, you make the script look like this:
+Kung , sa halip ay ang ``pick'' o ``edit'', na iyong itinuring sa halip na ``squash'', ang Git naaangkop sa bawat pagbabago at paglipat ng direkta bago ito at gumagawa ng merge na commit na mansahe na magkasama.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]: Kung , - remove space

File: book/07-git-tools/sections/rewriting-history.asc [hunk 19]
```diff
@@ -236,7 +236,7 @@ updated README formatting and added blame
 added cat-file
 ----
 
-When you save that, you have a single commit that introduces the changes of all three previous commits.
+Kapag ikaw ay nag-save doon, Kapag na-save na ito, mayroon kang isang solong commit na ipinakilala sa mga pagbabago sa lahat ng tatlong nakaraan na mga commit.
```
Comment (heywens) [inline book/07-git-tools/sections/rewriting-history.asc]:
~~~Kapag ikaw ay nag-save doon,~~ Kapag na-save na ito

**Review by heywens — COMMENTED** [review]
Hi Sir @robin-ho, done reviewing your contribution. I have some minor suggestions though. Kindly check.

Thanks.

File: book/07-git-tools/sections/rewriting-history.asc (hunk 1 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 2 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 3 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 4 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 5 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 6 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 7 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 8 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 9 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 10 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 11 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 12 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 13 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 14 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 15 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 16 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 17 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 18 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

File: book/07-git-tools/sections/rewriting-history.asc (hunk 19 above)
Comment (mind-blast) [reply book/07-git-tools/sections/rewriting-history.asc]: done change

**Comment by mind-blast** [comment]
done change all suggested words. Thank you for the proof read.
